controller/manager/manager.go

```diff
@@ -772,12 +814,36 @@ func (m DefaultManager) Registry(name string) (*shipyard.Registry, error) {
 		return nil, err
 	}
 
-	registry, err := shipyard.NewRegistry(reg.ID, reg.Name, reg.Addr)
+	if err := reg.InitRegistryClient(); err != nil {
+		log.Errorf("%s", err.Error())
+		return reg, err
+	}
+
+	return reg, nil
+}
+
+func (m DefaultManager) RegistryByAddress(addr string) (*shipyard.Registry, error) {
+	res, err := r.Table(tblNameRegistries).Filter(map[string]string{"addr": addr}).Run(m.session)
+	defer res.Close()
 	if err != nil {
 		return nil, err
 	}
+	if res.IsNil() {
+		log.Debugf("its nil!! it found nothing")
```

Comment:
can this be removed or more descriptive?  this by itself in the logs is not very helpful.